functions/src/routes/experiences/getAttendees.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable promise/always-return */
+/* eslint-disable promise/no-nesting */
+const admin = require("firebase-admin");
+
+const db = admin.firestore();
+
+const getAttendees = (req, res) => {
+	const {id: userId} = req.payload;
+	// const userId = "kjjxTGXIvcKy2u4dUe1f";
+	const {experienceId} = req.query;
+	const experienceRef = db.collection("experiences").doc(experienceId);
+	const userRef = db.collection("users").doc(userId);
+
+	return userRef.get()
```

Comment:
The former code on this line was a Promise.all that get's the signed user along with the experience at once, any particular reason for the extra .then??

Reply:
I saw that, but that was where the issue was coming from when I ran the endpoint. For some reasons, it was throwing "Promise is not iterable". You remember you were trying to loop through the result, so that's where it got mixed up. So I just went down an easier route.

Reply:
Ooh Interesting. JS can be irritating at times